metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/MetadataImporterMainApplication.java

```diff
@@ -45,22 +44,23 @@
 
     private ObjectsImporter objectsImporter;
 
-    private StudyRepository studyRepository;
-
-    public MetadataImporterMainApplication(ObjectsImporter objectsImporter, StudyRepository studyRepository) {
+    public MetadataImporterMainApplication(ObjectsImporter objectsImporter) {
         this.objectsImporter = objectsImporter;
-        this.studyRepository = studyRepository;
     }
 
     public static void main(String[] args) throws Exception {
         SpringApplication.run(MetadataImporterMainApplication.class, args);
+
     }
 
     @Override
     public void run(ApplicationArguments applicationArguments) throws Exception {
```

Comment:
Add a method comment describing why the starting points when importing from database and from API are different.